airbyte-cdk/python/connector_builder/connector_builder_handler.py

```diff
@@ -58,5 +56,42 @@ def resolve_manifest(source: ManifestDeclarativeSource) -> AirbyteMessage:
         return error.as_airbyte_message()
 
 
+def list_streams(source: ManifestDeclarativeSource, config: Dict[str, Any]) -> AirbyteMessage:
+    try:
+        streams = list(
```

Comment:
Instead of using a combination of mapping a function over the elements and putting it back into a list, it might be simpler and a little more idiomatic to just use list comprehension. Since `_get_http_streams()` returns a list, we're effectively doing a list -> list transformation.

```
[{"name": http_stream.name, "url": urljoin(http_stream.url_base, http_stream.path())} for http_stream in _get_http_streams(source, config)]

```